src/handlers/sendEmailRequestToNotify.ts

```diff
@@ -0,0 +1,41 @@
+import { APIGatewayProxyEvent } from 'aws-lambda'
+import { NotifyClient } from 'notifications-node-client'
+import { retrieveNotifySecrets } from '../secrets/retrieveNotifySecrets'
+import { CustomAxiosResponse } from '../types/notify/axiosResponse'
+import { tryParseJSON } from '../utils/helpers'
+
+// event type is a placeholder
+export const handler = async (event: APIGatewayProxyEvent) => {
+  if (!event.body) {
+    console.log('Could not find event body. An email has not been sent')
+    return
+  }
+
+  const requestDetails = tryParseJSON(event.body)
+  const secrets = await retrieveNotifySecrets()
```

Comment:
I'd like to see something here to check that the required fields are set in the SQS event body (and throw an error if they're not). That way, we don't get weird unexpected errors further down the line from things being missing.